[ao] fixing public v private for fx.backend_config_utils.py

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #86037

Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

HDCharles committed:
[ao] fixing public v private for fx.backend_config_utils.py

Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on torch/ao/quantization/fx/backend_config_utils.py:
```diff
@@ -137,6 +137,7 @@ def get_native_quant_patterns(additional_quant_patterns: Dict[Pattern, Quantizer
 get_pattern_to_quantize_handlers.__module__ = "torch.ao.quantization.fx.backend_config_utils"
 
 __all__ = [
+    "get_quantize_handler_cls",
```
this is a helper function I think, we can probably make it private

HDCharles commented:
@pytorchmergebot merge

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles committed:
Update on "[ao] fixing public v private for fx.backend_config_utils.py"


Summary: just added a missing function to __all__

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@pytorchmergebot merge
